README.md

```diff
@@ -55,13 +54,53 @@ We have some docs to make it easier to get you started:
 
 ## Deployment
 
-The website is built and deployed on [Netlify](https://netlify.com/) to our production environment ([https://yld.io](https://yld.io)) when:
+The website and lambda are built (`yarn build`) and deployed on [Netlify](https://netlify.com/) to our production environment ([https://yld.io](https://yld.io)) when:
 
 - a new commit is pushed to our **production** (`master`) branch, read more [here](https://www.netlify.com/docs/continuous-deployment/);
 - the Contentful data is updated (via _webhook_).
 
 Also, for each Pull Request that's open, a [Deploy Preview](https://www.netlify.com/blog/2016/07/20/introducing-deploy-previews-in-netlify/) is created, allowing for that branch to be tested and shared amongst stakeholders.
 
+## Automated deployments
+
+### 📬 Webhook automated deployments
+
+We are using Netlify webhooks/build hooks to automate new builds when services we use update. Each service has its own webhook url set up in Netlify.
+
+Keep in mind if you need to urgently alter any of the automated builds but don't have access to the services below, it's possible to just remove the webhook url from Netlify and stop it temporarily!
+
+#### Medium
+
+Zapier is subscribed to the yld engineering medium account via an RSS feeds, it checks every hour for new content. Upon new content zapier makes a post request to Netlify to trigger a fresh build.
+
+The account is registered under `apis@yld.io`, for access speak with Carlos Vilhena.
+
+### ƛ Netlify lambda automated deployments
+
+Utilising Netlify's [functions](https://www.netlify.com/docs/functions/).
+
+#### Meetup
+
+Local development requires:
+
+`MEETUP_KEY` - Key for the lambda to get content from Contentful. Available from Contentful API Keys Settings.
+
+`CMS_CRUD` - A personal access token generated from your Contentful account settings to allow writing to the yld Contentful space. Anyone with a Contentful account can generate one of these. The token used in production is registered to the `apis@yld.io` Contentful account.
+
+`./src/functions/meetup.js`
+
+Gets events from meetup.com and writes to Contentful which then triggers a deployment via the Contentful -> Netlify webhook.
```

Comment:
Maybe pop this line at the top, just to provide context first, before then giving details about how to use (i.e. env variables) afterwards?

Reply:
I purposefully put these envs and path to function above the explanation as it mimics the same style for function documentation.